### ISSUE TITLE
Extension autoselects wrong configuration?

### ISSUE THREAD

**Issue by JonathanFry3** [microsoft/vscode-cpptools#6281]
Install VS Code on a Linux guest (Ubuntu 20.04) on VMWare hosted on Windows 10.  
Install the C/C++ 1.0.1 extension.  
Edit the extension configuration.  The GUI says Linux, Mac and Windows are special configurations that will be autoselected when the extension is installed on that OS, but the configuration it autoselected is WSL.  Linux can be added and selected manually.

**Comment by sean-mcmanus** [comment]
What? We don't have a WSL default configuration. Can you provide a screenshot?

**Comment by JonathanFry3** [comment]
Sure.  In the list of available configurations at the top, I added "Linux".  It was not there after install.
![VS Code Screenshot](https://user-images.githubusercontent.com/66705367/95580811-f1a26c80-0a05-11eb-9d2d-96d289171afb.png)

**Comment by sean-mcmanus** [comment]
Did you create the WSL configuration yourself? What does it show as the default config name if you delete c_cpp_properties?

**Comment by JonathanFry3** [comment]
I probably did create the WSL config myself.  I worked in WSL for a while before its limitations drove me out, and I guess I got that config as part of my github repository.  If I rename c_cpp_properties, the default is Linux.  Sorry to bother you.    On Friday, October 9, 2020, 01:37:43 PM EDT, Sean McManus <notifications@github.com> wrote:  
 
 


Did you create the WSL configuration yourself? What does it show as the default config name if you delete c_cpp_properties?
